### PR TITLE
Feature/#159 fix terminology

### LINKED ISSUE #159: Unify terminology
Besides "golden master"—the term we agreed on—we use different words to describe the same thing. For instance:

* `RecheckStateFileProviderImpl`: "state file"
* `NoRecheckFileActionReplayResult`: "recheck file"
* …

I think it would be helpful for both the user and us to unify the terminology and consistently use the term "golden master".

### PR DESCRIPTION
**Pull request by enesoezel**
resolves#159

### CODE REVIEW

File: src/main/java/de/retest/recheck/NoGoldenMasterActionReplayResult.java [hunk 1]
```diff
@@ -20,9 +20,9 @@
 
 public class NoGoldenMasterActionReplayResult extends ActionReplayResult {
 
-	private static final String MSG_SHORT = "No recheck file found.";
+	private static final String MSG_SHORT = "No 'Golden Master' found.";
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/NoGoldenMasterActionReplayResult.java]: Needs no (single) quotes here and elsewhere.

**Review by beatngu13 — CHANGES_REQUESTED** [review]
Looks good. I just wonder if this breaks compatability beyond our own projects, i.e., external API? @modulo11 @roesslerj

**Comment by diba1013** [comment]
Just a reminder: Documentation

**Comment by beatngu13** [comment]
> Just a reminder: Documentation

We already use Golden Master there, but we probably should explain what is the difference between a Golden Master and a (SUT) state.

**Review by beatngu13 — APPROVED** [review]

**Pull request merged**